### ISSUE TITLE
Add support for code hover/select of method name with spaces, etc.

### ISSUE THREAD

**Issue by eric-milles** [groovy/groovy-eclipse#641]
Consider the following:
```groovy
def "some method name for whatever reason"() {
}
```

Code hover/select (F2/F3) should treat entrie method name string as one entity.  Currently selection has been disabled in `org.codehaus.groovy.eclipse.codebrowsing.requestor.CodeSelectHelper.isStringLiteral(ASTNode, char[], int, int)`.  With the extra clause removed from "isStringLiteral", selection takes place on individual words of the method name only.

**Comment by mauromol** [comment]
Does really Groovy support something like that? :-O

**Comment by eric-milles** [comment]
Yes indeed.  Any walkthrough of the Spock framework for testing shows method names in this fashion.  I think testing is the primary use case for the feature.  Although a.getAt("b"), a["b"] and a."b" all work fine in case you did this in general practice.
